lib/rspec/support/spec/shell_out.rb

```diff
@@ -62,13 +45,6 @@ def run_ruby_with_current_load_path(ruby_command, *flags)
           %r{jruby-\d\.\d\.\d\.\d/lib/ruby/stdlib/rubygems},
           # This is required for windows for some reason
           %r{lib/bundler/rubygems},
-          # This is a JRuby file that generates warnings on 9.0.3.0
-          %r{lib/ruby/stdlib/jar},
-          # This is a JRuby file that generates warnings on 9.1.7.0
-          %r{org/jruby/RubyKernel\.java},
-          # This is a JRuby gem that generates warnings on 9.1.7.0
```

Comment:
I remember @headius mentioned that for the end-users there is no good excuse for not using the latest 9.2 instead of any other JRuby version. Did I understand/remember this correctly?